Dockerfile

```diff
@@ -1,31 +1,32 @@
 # ================ PYTHON
-FROM python:3.7
+FROM python:3.7-alpine
 
 # System setup:
-RUN apt-get update && apt-get install -y gettext redis-tools ca-certificates && apt-get clean
+RUN apk update && \
+    apk add --no-cache gettext redis ca-certificates
```

Comment:
Oh hey, if we're installing `ca-certificates` and `curl` here in the base image, can I remove those from the `install-dev-tools.sh` script?

Reply:
Yeah, but all of `install-dev-tools.sh` will have to be changed to use Alpine's `apk` rather than Debian/Ubuntu's `apt-get`, too.

And we're not installing `curl` here, unless I'm missing something?

Reply:
We're installing curl below... but okay, maybe not worth converting to `apk` unless there's another good reason to.

Reply:
Well, if we use Alpine, the image will be a lot smaller, but we'll have to use `apk`, because that's the package manager on Alpine.

Reply:
@wlonk I see. You'll go ahead and make the required corresponding changes to the install-dev-tools script then?

Reply:
Yes, though I wanted to hold off until it seemed to make sense given:
> Until VSCode dev in an Alpine container moves from experimental to supported (see [here](https://code.visualstudio.com/docs/remote/containers#_getting-started)), we should probably not merge this.